Source/DafnyCore/Generic/Reporting.cs

```diff
@@ -46,14 +44,6 @@ public abstract class ErrorReporter {
     public virtual void Error(MessageSource source, string errorId, IToken tok, string msg) {
       Contract.Requires(tok != null);
       Contract.Requires(msg != null);
-      // TODO move this out of ErrorReporter together with OuterModule. Let Program have a list of FileModuleDefinitions as constructor input and create its default module there.
-      if (tok.FromIncludeDirective(OuterModule) && OuterModule != null) {
-        var include = OuterModule.Includes.First(i => i.IncludedFilename == tok.Uri);
-        if (!include.ErrorReported) {
-          Message(source, ErrorLevel.Error, null, include.tok, "the included file " + Path.GetFileName(tok.ActualFilename) + " contains error(s)");
-          include.ErrorReported = true;
-        }
-      }
```

Comment:
I'm all for this change, but we should be sure to mention this in the CHANGELOG since I know there are users (including myself) who are accustomed to the behavior, and might be confused if we change it silently.